### PR TITLE
skipper-canary: update logs destination

### PR DESCRIPTION
**Pull request by MustafaSaber**
provide pod name as env variable

### CODE REVIEW

**Comment by RomanZavodskikh** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
:+1:

File: cluster/manifests/skipper-canary-controller/canary-cronjob.yaml [hunk 1]
```diff
@@ -18,6 +18,10 @@ spec:
           labels:
             application: skipper-ingress
             component: canary
+        annotations:
+          kubernetes-log-watcher/scalyr-parser: |
+              [{"container": "controller", "parser": "keyValue"}]
+          logging/destination: "{{ .Cluster.ConfigItems.log_destination_both }}"
```
Comment (demonCoder95) [inline cluster/manifests/skipper-canary-controller/canary-cronjob.yaml:24]: @MustafaSaber  I think the indent needs to align with `labels` and not `metadata`. CLM fails to apply the changes: https://app.eu.scalyr.com/y/YiQ9PHLSFdM 
```
strict decoding error: unknown field \"spec.jobTemplate.spec.template.annotations
```

Comment (MustafaSaber) [reply cluster/manifests/skipper-canary-controller/canary-cronjob.yaml:24]: 🤦🏻‍♂️🤦🏻‍♂️

**Pull request merged**